src/main/java/core/basesyntax/HelloWorld.java

```diff
@@ -4,5 +4,20 @@
  * Feel free to remove this class and create your own.
  */
 public class HelloWorld {
+    public static final int COUNT_FIGURES = 6;
```

Comment:
Consider using a named constant for the division `COUNT_FIGURES/2` in the loop condition to improve code readability and maintainability.